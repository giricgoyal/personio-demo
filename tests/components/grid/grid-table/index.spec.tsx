import React from 'react'
import GridTable from 'src/components/grid/grid-table'
import { render } from 'tests/test-utils'

describe('components/grid/grid-table', () => {
    let container

    beforeEach(() => {
        ;({ container } = render(
            <GridTable columns={2}>
                <div></div>
            </GridTable>,
        ))
    })

    test('should render and match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
