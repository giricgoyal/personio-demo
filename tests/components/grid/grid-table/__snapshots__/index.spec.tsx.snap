// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/grid/grid-table should render and match snapshot 1`] = `
<div>
  <div
    class="grid__table grid__table--grid-2"
  >
    <div />
  </div>
</div>
`;
